File: app/src/main/java/ru/psu/studyit/view/activities/lab/Fragment_materials.java
//@author Дегтяникова Дарья
package ru.psu.studyit.view.activities.lab;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.fragment.app.Fragment;

import ru.psu.studyit.R;

public class Fragment_materials extends Fragment {
    public Fragment_materials() {
// Required empty public constructor
    }

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
// Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_lab_materials, container, false);

    }
}

File: app/src/main/java/ru/psu/studyit/view/activities/lab/Fragment_materials.kt
//@author Дегтяникова Дарья
package ru.psu.studyit.view.activities.lab

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import androidx.fragment.app.Fragment

import ru.psu.studyit.R

class Fragment_materials : Fragment()
{

    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)
    }

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View?
    {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_lab_materials, container, false)

    }
}// Required empty public constructor
